quantum/quantum.c

```diff
@@ -1151,22 +1162,22 @@ void matrix_scan_quantum() {
 #    endif
 
 #    ifndef BACKLIGHT_ON_STATE
-#        define BACKLIGHT_ON_STATE 0
+#        define BACKLIGHT_ON_STATE 1
```

Comment:
how is backward compatibility handled here? If people have their own keyboard definitions downstream this will break their backlight. Any chance we can warn them or help them in some way?